openbb_platform/core/openbb_core/provider/standard_models/options_snapshots.py

```diff
@@ -27,53 +27,52 @@ class OptionsSnapshotsData(Data):
     expiration: List[dateType] = Field(
         description="Expiration date of the options contract."
     )
-    dte: List[Union[int, None]] = Field(
-        default_factory=list,
+    dte: Optional[List[int]] = Field(
+        default=None,
         description="Number of days to expiration of the options contract.",
     )
     strike: List[float] = Field(
         description="Strike price of the options contract.",
         json_schema_extra={"x-unit_measurement": "currency"},
     )
     option_type: List[str] = Field(description="The type of option.")
-    volume: List[Union[int, None]] = Field(
-        default_factory=list,
+    volume: Optional[List[int]] = Field(
         description=DATA_DESCRIPTIONS.get("volume", ""),
     )
-    open_interest: List[Union[int, None]] = Field(
-        default_factory=list,
+    open_interest: Optional[List[int]] = Field(
+        default=None,
         description="Open interest at the time.",
     )
-    last_price: List[Union[float, None]] = Field(
-        default_factory=list,
+    last_price: Optional[List[float]] = Field(
+        default=None,
         description="Last trade price at the time.",
         json_schema_extra={"x-unit_measurement": "currency"},
     )
-    last_size: List[Union[int, None]] = Field(
-        default_factory=list,
+    last_size: Optional[List[int]] = Field(
+        default=None,
         description="Lot size of the last trade.",
     )
-    last_timestamp: List[Union[datetime, None]] = Field(
-        default_factory=list,
+    last_timestamp: Optional[List[datetime]] = Field(
```

Comment:
These need to be defined as a `List[Union[Type, None]` because the "Field" is NOT Optional. Values within the array can be a Union of [Int, None]. `None` is a valid Type.

Reply:
Thanks for clarifying, now I understand. Closing the PR then.